integration/src/test/resources/templates/kotlin-multiplatform/build.gradle

```diff
@@ -2,15 +2,9 @@ import org.jetbrains.kotlin.konan.target.KonanTarget
 import org.jetbrains.kotlin.konan.target.HostManager
 
 kotlin {
-    jvm {
-        compilations.all {
-            kotlinOptions.jvmTarget = '1.8'
-        }
-    }
-
-    js {
-        nodejs()
-    }
+    jvm()
+    js('jsIr', IR) { nodejs() }
+    js('jsLegacy', LEGACY) { nodejs() }
```

Comment:
Why not simple `js {nodejs()}`. `jsLegacy` could be confusing as new backend still not stable and not default. It's also inconsistent with kotlin gradle plugin dsl, where `js  {}` is used

Reply:
it's template for integration tests, to test for both LEGACY and IR
In example, yes, this could be changed to `js`.
And then, somewhere should be noted, that `IR` is also supported. 
And if users will use `kotlin.js.compiler=both` or `js(BOTH)` - this will not work, as `BOTH` doesn't support generating  executables

Reply:
@qurbonzoda @anton-bannykh what do you think?

Reply:
Indeed `js('jsLegacy', LEGACY)` could be just `js {}`, but I think for the test project used in integration tests it's not that critical. It makes sense to fix it in `kotlin-multiplatform` example project though.

I will update README with the IR support information when it gets released.

Reply:
`jsLagacy` is now replaced to `js`: https://github.com/Kotlin/kotlinx-benchmark/pull/77